main.go

```diff
@@ -25,6 +25,7 @@ import (
 const defaultConfig = `
 amqp:
   uri: amqp://guest:guest@rabbit:5672/
+  queue_prefix: "templeton"
```

Comment:
Heh, I knew I shouldn't have left semi-contradictory messages. If we want the default to be the queue name prior to this PR, this default should be the empty string.